README.md

```diff
@@ -51,6 +51,16 @@ python manage.py migrate
 python manage.py createsuperuser --username=admin
 ```
 
+Postgres.app is a standalone PostgreSQL server that includes the PostGIS extension. You will also need to install gdal and libgeoip with Homebrew:
```

Comment:
:neckbeard: _nitpick_ `libgeoip` is mentioned, but only `postgis` and `gdal` are installed below. Is that intentional?

🙋‍♂️ Would it be worth mentioning why `gdal` and `libgeoip` need to be installed?